lib/y2logsstep.pm

```diff
@@ -243,11 +243,15 @@ sub accept_license {
 
 sub verify_license_translations {
     return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
-    for my $language (split(/,/, get_var('EULA_LANGUAGES')), 'english-us') {
+    my $current_lang = 'english-us';
+    for my $lang (split(/,/, get_var('EULA_LANGUAGES')), 'english-us') {
         wait_screen_change { send_key 'alt-l' };
-        send_key 'home';
-        send_key_until_needlematch("license-language-selected-$language", 'down', 60, 3);
-        assert_screen "license-content-$language";    # needs wait for loading content
+        assert_and_click "license-language-selected-$current_lang";
+        wait_screen_change { type_string(substr($lang, 0, 1)) };
+        send_key_until_needlematch("license-language-selected-dropbox-$lang", 'down', 60);
```

Comment:
I would still keep 3 seconds for the timeout, as 1s may not be enough and we will end up in the same scenario when we press down multiple times skipping some languages.

Reply:
I'd like to keep it that way, it should be super fast as there is not loading of anything like in the other screen, which make it really problematic, I experienced with previous code 3 seconds from the previous language to korean to korean, and I cannot imaging in some slow worker what could be. Here moving down we should be playing safe, otherwise typing the first letter would not be a good idea neither if there is some background loading but my assumption is that there is not.

Reply:
Alright, no problem at all, let's try it then, as anyway it will affect single test suite at most.